<launch>
	<!-- vim: ft=xml -->
	<!-- example node launch file-->

	<arg name="shm_unit" default="2" />
	<arg name="time_ref" default="/time_ref" />

	<node pkg="ntpd_driver" type="shm_driver" name="ntpd_shm" respawn="true" respawn_delay="10" clear_params="true" output="screen">
		<param name="shm_unit" value="$(arg shm_unit)" />
		<remap from="~/time_ref" to="$(arg time_ref)" />
	</node>
</launch>
